scenes/water.scn

```diff
@@ -49,15 +46,8 @@
         0,
         0
       ],
+      "scale": [100, 1, 100],
       "start_url": "/practice/water-main2/"
-    },
-    {
-      "position": [
-        0,
-        0,
-        0
-      ],
-      "start_url": "/practice/dual-contouring-terrain-main/"
```

Comment:
do we need these changes in pr?

Reply:
No, we don't need these changes in water.scn. I made them to have more space for swimming while testing. Will remove them.